sdk/schemaregistry/Azure.Data.SchemaRegistry/README.md

```diff
@@ -103,18 +103,18 @@ string schemaContent = @"
     ]
 }";
 
-Response<SchemaProperties> schemaProperties = client.RegisterSchema(groupName, schemaName, schemaType, schemaContent);
+Response<SchemaProperties> schemaProperties = client.RegisterSchema(groupName, name, content, type);
 ```
 
 ### Retrieve a schema ID
 
 Retrieve a previously registered schema ID from the Azure Schema Registry. When looking up the schema Id, the `SchemaProperties` will be cached in the `SchemaRegistryClient` instance, so that subsequent requests for this schema do not need to go the service.
 
 ```C# Snippet:SchemaRegistryRetrieveSchemaId
```

Comment:
nit:  Should this be updated to something like:  "C# Snippet:SchemaRegistryRetrieveSchemaProperties" ?

Reply:
I think it probably doesn't matter especially considering the section is talking about getting a schema ID.